homeassistant/components/deconz/device_trigger.py

```diff
@@ -129,6 +146,55 @@
     (CONF_ROTATED, CONF_RIGHT): 2002,
 }
 
+AQARA_CUBE_MODEL = "lumi.sensor_cube"
```

Comment:
The Cube remote is defined, but these variables are unused. Add it to `REMOTES`?

Reply:
As soon as you mentioned it on Discord I realised this :). Thanks!